botocore/client.py

```diff
@@ -14,19 +14,16 @@
 import functools
 import logging
 
-from botocore.model import ServiceModel
-from botocore.exceptions import DataNotFoundError
+import botocore.serialize
+import botocore.validate
+from botocore import credentials, retryhandler, translate, waiter, xform_name
+from botocore.endpoint import EndpointCreator
+from botocore.exceptions import ClientError, DataNotFoundError
 from botocore.exceptions import OperationNotPageableError
-from botocore.exceptions import ClientError
-from botocore import waiter
-from botocore import xform_name
+from botocore.model import ServiceModel
 from botocore.paginate import Paginator
-from botocore.utils import CachedProperty
-import botocore.validate
-import botocore.serialize
-from botocore import credentials
 from botocore.signers import RequestSigner
-from botocore.endpoint import EndpointCreator
+from botocore.utils import CachedProperty
```

Comment:
Note opposed to this change, but do note that alphabetical order is not mentioned anywhere in pep8 imports (https://www.python.org/dev/peps/pep-0008/#imports).